project/config/pytest.ini

```diff
@@ -13,3 +13,6 @@ addopts =
   --cov
   --cov-append
   --cov-config config/coverage.ini
+testpaths =
+    tests
+    integration
```

Comment:
What's that for? You split your tests into two folders? Maybe you can convince me I should too? Otherwise I'll ask you to revert this!

Reply:
I removed the integration folder as I'm not setup for that yet. but i did add this so its explicit as that's how its setup already.
https://docs.pytest.org/en/stable/customize.html#pytest-ini